README.md

```diff
@@ -107,28 +107,6 @@ On your guest system, you have to adjust your DNS settings like mentioned before
 
 #### Help - My Docker Setup is not working as expected!
 
-##### One of the images can't be built:
-You might see the following in your console after typing `docker-compose up` for the first time (which will initially build the containers):
-
-```
-[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
-[warn] 	::          UNRESOLVED DEPENDENCIES         ::
-[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
-[warn] 	:: pricewars-utils#pricewars-utils_2.11;0.1-SNAPSHOT: not found
-[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
-[warn]
-[warn] 	Note: Unresolved dependencies path:
-[warn] 		pricewars-utils:pricewars-utils_2.11:0.1-SNAPSHOT
-[warn] 		  +- flinkutils:flinkutils_2.11:1.0
-```
-
-Most probably, the utils submodule was not cloned correctly. Check that `analytics\utils` is not empty. Within the `analytics` folder, execute the following command:
```

Comment:
I have never encountered this problem with `analytics/utils`, but are you sure we should remove this hint?

Reply:
This was an error that could happen when `analytics\utils` was a sub-submodule.
This is no longer the case. (It's now integrated into the analytics submodule)